docs/src/details/integration/renderers/espressif_ppa.rst

```diff
@@ -0,0 +1,8 @@
+======================================
+Espressif Pixel Processing Accelerator
+======================================
+
```

Comment:
Please a little more info here too:
- Supported devices
- How to use
- What is accelerated
- The disclaimer about the performance